src/SourceLink.AzureDevOpsServer.Git/build/Microsoft.SourceLink.AzureDevOpsServer.Git.targets

```diff
@@ -63,7 +63,7 @@
     </Microsoft.SourceLink.AzureDevOpsServer.Git.TranslateRepositoryUrls>
 
     <ItemGroup>
-      <SourceRoot Remove="@(SourceRoot)"/>
+      <SourceRoot Remove="@(SourceRoot)" Condition="'@(_TranslatedSourceRoot)' != ''"/>
```

Comment:
>" Condition="'@(_TranslatedSourceRoot)' != ''"/> [](start = 39, length = 48)

This is a product change. Is there an issue filed that this is fixing? We should also have a test covering this if this is indeed a product bug.

Reply:
@crummel do you have any extra info on this patch?

Reply:
I remember that `TranslateRepositoryUrls` was returning an empty list for `TranslatedSourceRoot`s.  Then with the removal of the original SourceRoots, we ended up with no SourceRoots at all, which caused the usual "no SourceRoots" error.  I'll take a look at why this was happening.

Reply:
Removed for now.

Reply:
My test build looks good without this, so I think it was a temporary issue.